Sodium/Box.swift

```diff
@@ -373,6 +373,11 @@ extension Box: KeyPairGenerator {
         public typealias SecretKey = Box.SecretKey
         public let publicKey: PublicKey
         public let secretKey: SecretKey
+
+        public init(publicKey: PublicKey, secretKey: SecretKey) {
```

Comment:
Isn't this initialiser identical to the [compiler provided one](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Initialization.html#//apple_ref/doc/uid/TP40014097-CH18-ID214)?